packages/react-router/modules/matchPath.js

```diff
@@ -30,7 +30,8 @@ const matchPath = (pathname, options = {}) => {
   if (typeof options === 'string')
     options = { path: options }
 
-  const { path, exact = false, strict = false } = options
+  const { exact = false, strict = false } = options
+  const path = options.path || options.from
```

Comment:
Seems out of place to me. The components define this API (from/to/path) not `matchPath`